person.rb

```diff
@@ -26,4 +27,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Same here,  This method supposes to add a new rental not include an already created one to an array, and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Please fix that.